packages/account/src/Components/poi/status/unsupported/detail-component.jsx

```diff
@@ -6,7 +6,7 @@ import { WS } from '@deriv/shared';
 import { UploadComplete } from '../upload-complete/upload-complete';
 import PoiUnsupportedFailed from 'Components/poi-unsupported-failed';
 import uploadFile from 'Components/file-uploader-container/upload-file';
-import OnfidoUpload from '../../../../Sections/Verification/ProofOfIdentity/onfido-sdk-view';
+import OnfidoUpload from '../../../../Sections/Verification/ProofOfIdentity/onfido-sdk-view-container';
```

Comment:
Can we use alias here?
```suggestion
import OnfidoUpload from 'Sections/Verification/ProofOfIdentity/onfido-sdk-view-container';
```

Reply:
Will make this change :)